website/content/docs/auth/kubernetes.mdx

```diff
@@ -101,9 +106,91 @@ management tool.
     For the complete list of configuration options, please see the [API
     documentation](/api/auth/kubernetes).
 
+### How to work with short-lived Kubernetes tokens
```

Comment:
I think this section would have better value to the reader at the top of the page where the flow is as follows. 

- Kuberneetes Auth Method
    - Note on JWT auth
- Explanation of different Auth methods & choosing whats right for you. 
- Configuration examples

Reply:
I want to keep the pattern of having quick reference setup commands highly visible at the top. However, I agree it would be good to draw attention to this earlier, so I've added a note to the opening paragraph and pulled all the documentation for Kubernetes 1.21 changes under one section.